services/device-registry-base/src/main/java/org/eclipse/hono/service/DeviceRegistryMetrics.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.hono.service;
+
+import org.eclipse.hono.service.metric.Metrics;
+import org.eclipse.hono.service.metric.NoopBasedMetrics;
+
+/**
+ * Metrics for the Device Registry service.
+ */
+public interface DeviceRegistryMetrics extends Metrics {
+
+    /**
+     * Metric key for total number of Tenants in the system.
+     */
+    String TOTAL_TENANTS_METRIC_KEY = "total.tenants";
```

Comment:
can we please move this to *MicrometerBasedMongoDbDeviceRegistryMetrics* and rename it according to the conventions we use for other meter names, i.e. using a `hono.` prefix, e.g. *hono.tenants.total*?